src/main/java/com/pinterest/secor/tools/ProgressMonitor.java

```diff
@@ -207,8 +207,14 @@ private void exportToStatsD(List<Stat> stats) {
                     assert committedOffset <= lastOffset: Long.toString(committedOffset) + " <= " +
                         lastOffset;
 
-                    long offsetLag = lastOffset - committedOffset;
-                    long timestampMillisLag = lastTimestampMillis - committedTimestampMillis;
+                    long offsetLag = 0L;
```

Comment:
Should the code just return around line 195?

If committed message is null, what will the value of lastMessage?  Will that also be null?  Otherwise secor would have another upload after the committed message.